etc/teams-for-linux.profile

```diff
@@ -0,0 +1,27 @@
+include disable-common.inc
+include disable-devel.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include globals.local
+
+caps.drop all
+netfilter
```

Comment:
Need it dbus access or can we add `nodbus`?

Reply:
It'll need dbus for notification ability. I'll add the other suggestions, and test them out on my install before updating the pull request. Thank you for the feedback, first time contributing to this, and I appreciate greatly the suggestions.